File: testers/openapi/java/jaxrs/testapi/api/CreatePetOnPetCreatedCallback.java
package testapi.api;

import testapi.model.Pet;

/**
 * Callback handler for createPet - OnPetCreated Runtime expression: {$request.body#/callbackUrl}
 */
public interface CreatePetOnPetCreatedCallback {
  /** Called when pet is created */
  Void onPetCreatedCallback(Pet body);
}

File: testers/openapi/kotlin/jaxrs/testapi/api/CreatePetOnPetCreatedCallback.kt
package testapi.api

import testapi.model.Pet

/** Callback handler for createPet - OnPetCreated
  * Runtime expression: {$request.body#/callbackUrl}
  */
interface CreatePetOnPetCreatedCallback {
  /** Called when pet is created */
  abstract fun onPetCreatedCallback(body: Pet): Unit
}